registry/client/auth/session_test.go

```diff
@@ -41,8 +42,8 @@ func testServerWithAuth(rrm testutil.RequestResponseMap, authenticate string, au
 	wrapper := &testAuthenticationWrapper{
 
 		headers: http.Header(map[string][]string{
-			"Docker-Distribution-API-Version": {"registry/2.0"},
-			"WWW-Authenticate":                {authenticate},
+			"X-API-Version":    {"registry/2.0"},
```

Comment:
Yes, the default header name has nothing to do with this package, using a generic name ensures we aren't hard coding the value.